README.md

```diff
@@ -1,53 +1,31 @@
 # Beluga
 
-![CI badge](https://github.com/Ekumen-OS/beluga/actions/workflows/ci_pipeline.yml/badge.svg?event=push)
+[![CI pipeline](https://github.com/Ekumen-OS/beluga/actions/workflows/ci_pipeline.yml/badge.svg?branch=main)](https://github.com/Ekumen-OS/beluga/actions/workflows/ci_pipeline.yml?query=branch:main)
+[![License Apache-2.0](https://img.shields.io/badge/license-Apache--2.0-blue.svg)](LICENSE)
 
 ## Overview
 
-Beluga is an extensible library with a ground-up implementation of the Monte Carlo Localization (MCL) family of estimation algorithms,
-including sensor and motion models.
-It also provides a _drop-in_ replacement for the `amcl` node used in [navigation2](https://github.com/ros-planning/navigation2).
+Beluga is an extensible C++ library with a ground-up implementation of the Monte Carlo Localization (MCL) family of estimation algorithms featuring:
 
-https://github.com/Ekumen-OS/beluga/assets/33042669/df2a5e61-3d97-4350-b171-71bb1a09a6d0
+- A modular design based on orthogonal components that can be merged together using the mixin pattern.
+- Good test coverage to prevent regressions and facilitate code improvements.
```

Comment:
I don't quite like "good" here, sounds subjective, but I don't have a better alternative. Maybe:
```suggestion
- Emphasis on the prevention of regressions and facilitation of code improvements through test coverage.
```

Reply:
Changed in https://github.com/Ekumen-OS/beluga/pull/186/commits/d2d26f656dd3d429b7c8cac938e4b63129eeb646.